app/script/storage/StorageService.js

```diff
@@ -260,6 +260,16 @@ z.storage.StorageService = class StorageService {
               this.db[StorageService.OBJECT_STORE.EVENTS].bulkPut(items);
             });
         });
+      this.db.version(14).stores(version_12)
+        .upgrade((transaction) => {
+          this.logger.warn('Database upgrade to version 14', transaction);
+          transaction[StorageService.OBJECT_STORE.EVENTS].toCollection()
+            .modify((event) => {
+              if (event.type === 'conversation.asset-meta') {
```

Comment:
Do we have a constant for `'conversation.asset-meta'`?

Reply:
No, this is outdated and deprecated. Choose the string on purpose as this will be the only place we are going to use it moving forward - and we won't break it by accidentally removing the constant elsewhere.